File: 程式語言 C V2答案檔/303.c
#include <stdio.h>
#include <stdlib.h>

int compute(int);

int main()
{
	int num;
	scanf("%d", &num);
	if (compute(num) == 1) { 
		printf("%d is a prime number\n", num);
	}
	else {
		printf("%d is not a prime number\n", num);
	} 
    return 0;
}

// ===================================================
// Please write your code in the specified function
// Do NOT change the function name, type of parameter
// ===================================================
int compute(int num)
{
	int i;
	for(i = 2 ; i < num ; i++) {
		if(num % i == 0) {
			return 0;
		}
	}
	return 1;
}

File: 物件導向程式語言 C# 7答案檔/303.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSA03
{
    class CSA03
    {
        static void Main(string[] args)
        {
    
            int[] numbers = new int[5];
            for (int i = 0; i <= 4; i++)
            {
                Int32.TryParse(Console.ReadLine(), out numbers[i]);
            }
            compute(numbers);
        }
        // ==================================================
        // Please write your code in the specified Method
        // Do NOT change the method name, type of parameter
        // ==================================================
        static void compute(int[] numbers)
        {
            int tmp;
            for (int i = 0; i < 4; i++)
            {
                for (int j = 0; j < 4; j++)
                {
                    if (numbers[j] > numbers[j + 1])
                    {
                        tmp = numbers[j];
                        numbers[j] = numbers[j + 1];
                        numbers[j + 1] = tmp;
                    }
                }
            }
            Console.Write(string.Join(" ", numbers));
            Console.ReadKey();
        }

    }
}
